docs/manufacturer/manufacturer-section.md

```diff
@@ -56,6 +56,8 @@ By calling the route you can create a new customer in tapio ecosystem.
 
 > `timeZone` expected [as windows standard](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/default-time-zones).
 
+> `companyVat` gets validated via [VIES](https://ec.europa.eu/taxation_customs/vies/#/vat-validation) for EU countries.
```

Comment:
Do you want to mention that we have also algorithms for other countries? Or is that no longer done in the future?

Reply:
I don't want to bloat the Docu with details for each other country because there are only a few others. Maybe we will add a dedicated route to receive the information for each country in the future via the API. For now i decided that the API response if a VAT validation fails for another country should be enough.